ocs_ci/ocs/ui/views.py

```diff
@@ -147,6 +147,10 @@
     "enable_taint_node": ('input[id="taint-nodes"]', By.CSS_SELECTOR),
     "gp2-csi_sc": ('a[id="gp2-csi-link"]', By.CSS_SELECTOR),
     "gp3-csi_sc": ('a[id="gp3-csi-link"]', By.CSS_SELECTOR),
+    "choose_local_storage_version": (
```

Comment:
Where this one is used?

Reply:
after the discussion , this should not be changed when 4.10 is GAed. So this has to be in condition

Reply:
done